docs/_advanced/authorization.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Authorization
+lang: en
+slug: authorization
+order: 5
+---
+
+<div class="section-content">
+Authorization is the process of determining which Slack credentials should be available while processing an incoming Slack event.
+
+Apps installed on a single workspace can simply pass their bot token into the `App` constructor using the `token` parameter. However, if your app will be installed on multiple workspaces, you have two options. The easier option is to use the built-in OAuth support. This will handle setting up OAuth routes and verifying state. Read the section on [authenticating with OAuth](#authenticating-oauth) for details.
+
+For a more custom solution, you can set the `authorize` parameter to a function upon `App` instantiation. The `authorize` function should return [an instance of `AuthorizeResult`](https://github.com/slackapi/bolt-python/blob/main/slack_bolt/authorization/authorization_result.py#L6), which contains information about who and where the event is coming from.
```

Comment:
>OK, to avoid this confusion, I will rename AuthorizationResult to AuthorizeResult in the next alpha version.

The class has been renamed in v0.6.3a0.